Fix some geometry conversion tools (#6891)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/vectorgeometry.rst

Generates a new layer based on an existing one, with a different type
of geometry.

The attribute table of the output layer is the same as the one of
the input layer.

> Not all conversions are possible. For instance, a line can be
> converted to a point, but a point cannot be converted to a
> line. A line can also be converted to a polygon.

Author: ```suggestion
Not all conversions are possible. For instance, a line layer
can be converted to a point layer, but a point layer cannot
be converted to a line layer.
```

> .. seealso:: :ref:`qgispolygonize`, :ref:`qgislinestopolygons`, :ref:`qgispolygonstolines`

Reviewer: ```suggestion
.. seealso:: :ref:`qgispolygonize`, :ref:`qgislinestopolygons`, :ref:`qgispolygonstolines`
  :ref:`qgispointstopath`
```
? Just because we state few lines above that points can be converted to lines with this alg, let's give them an alg that can do that

Generates a new layer based on an existing one, with a different type
of geometry.

The attribute table of the output layer is the same as the one of
the input layer.

Not all conversions are possible. For instance, a line layer
can be converted to a point layer, but a point layer cannot
be converted to a line layer.

> .. seealso:: :ref:`qgispolygonize`, :ref:`qgislinestopolygons`, :ref:`qgispolygonstolines`
>   :ref:`qgispointstopath`

Author: ```suggestion
.. seealso:: :ref:`qgispolygonize`, :ref:`qgislinestopolygons`, :ref:`qgispolygonstolines`,
   :ref:`qgispointstopath`
```
